Stratis.Bitcoin/Utilities/DBreezeSerializer.cs

```diff
@@ -0,0 +1,122 @@
+﻿using DBreeze.Utils;
+using NBitcoin;
+using NBitcoin.BitcoinCore;
+using System;
+using System.Linq;
+
+namespace Stratis.Bitcoin.Utilities
+{
+    /// <summary>
+    /// Implementation of serialization and deserialization of objects that go into the DBreeze database.
+    /// </summary>
+    public class DBreezeSerializer
+    {
+        /// <summary>
+        /// Initializes custom serializers for DBreeze engine.
+        /// </summary>
+        public void Initialize()
+        {
+            CustomSerializator.ByteArraySerializator = Serializer;
```

Comment:
`this.Serializer`

Reply:
damn, always at least one instance i forget :) fixed